packages/fileutils/fileutils.0.6.5/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+maintainer: "Sylvain Le Gall <sylvain+ocaml@le-gall.net>"
+authors: [ "Sylvain Le Gall" ]
+homepage: "https://github.com/gildor478/ocaml-fileutils"
+license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
+dev-repo: "git+https://github.com/gildor478/ocaml-fileutils.git"
+bug-reports: "https://github.com/gildor478/ocaml-fileutils/issues"
+doc: "https://gildor478.github.io/ocaml-fileutils/"
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+depends: [
+  "ocaml" {>= "4.14"}
+  "base-unix"
+  "stdlib-shims"
```

Comment:
Are stdlib-shims and seq needed for OCaml 4.14? I think they are needed only for very old OCaml versions?

Reply:
I have removed them.